doc/cephadm/services/osd.rst

```diff
@@ -84,6 +84,39 @@ information about interacting with these LEDs, refer to :ref:`devices`.
     The current release of `libstoragemgmt`_ (1.8.8) supports SCSI, SAS, and SATA based
     local disks only. There is no official support for NVMe devices (PCIe)
 
+Retrieve Exact Size of Block Devices
+====================================
+
+Run a command of the following form to discover the exact size of a block
+device. The value returned here is used by the orchestrator when comparing high
+and low values:
+
+.. prompt:: bash #
+
+   cephadm shell ceph-volume inventory </dev/sda> --format json | jq .sys_api.human_readable_size
+
+The exact size in GB is the size reported in TB, multiplied by 1000.
```

Comment:
Most sizes in Ceph are in base 2 units, do we believe these to be base-10?

Reply:
I suppose that this is a matter to be litigated by people better versed in these matters of binary-versus-decimal than I. In making this pull request, I am relying on the in-the-field information provided here: https://tracker.ceph.com/issues/66754

(Of course every bit of our documentation is subject to alteration, so this base-ten claim can be altered at will. Anyway, I'm going to merge and backport this commit.)